jobs/build/rhcos/Jenkinsfile

```diff
@@ -25,10 +25,20 @@ node {
                     commonlib.ocpVersionParam('BUILD_VERSION', '4'),
                     string(
                         name: 'ARCHES',
-                        description: 'Comma delimited list of arches to target The RHCOS architecture to target.',
+                        description: 'List of architectures to build for RHCOS (ignored for multi pipeline).',
                         defaultValue: commonlib.brewArches.join(','),
                         trim: true,
                     ),
+                    booleanParam(
+                        name: 'IGNORE_EXISTING',
+                        description: '(Multi pipeline only) Create a new build even if one is already in progress.',
+                        defaultValue: false,
+                    ),
+                    booleanParam(
+                        name: 'BUILD_ALWAYS',
```

Comment:
Why not 'force'?
Also does this include "ignore_existing" implicitly ? I would assume it does
Would we have cases where we use "ignore_existing" and not "force"?

Reply:
> Why not 'force'?

Exactly because of this assumption :)

`FORCE` implies "ignore conditionals, just get me a new build". I don't want that hammer here. Also it implies ignoring the mutex in this job itself, which I don't ever want to do.

We need `BUILD_ALWAYS` when latest builds do not have all arches complete, or contents are not consistent. Current RHCOS pipeline logic will not actually rebuild for this reason (build will run and abort early) because no change is needed to x86. So we want our automation to ignore that and have a new build anyway.  Except we would not want to do this if the current pipeline is simply waiting on arch builds to finish, or if there is already another build in progress (possibly a human started it, possibly a previous run started it and then died).  So I intend automation to run `BUILD_ALWAYS` without `IGNORE_EXISTING` to first check if there are any running builds, and then force a build if not. This is to keep FORCEd runs from piling up. Admittedly probably needs a better name, since it _won't_ always trigger a build.

`IGNORE_EXISTING` is more intended for humans. (I called it `--skippable` in the params to invert the logic... likely needs more naming thought too.) Using our superior situational awareness we may see that any current runs are irrelevant to our purposes and we really want to force a new one ASAP. This will allow us to do so within the context of the mutex (waiting for the build we kick off) preventing automation from kicking off more runs.

Reply:
Agreed! It's good to have that button for when we might need it